aten/src/ATen/CMakeLists.txt

```diff
@@ -18,6 +18,9 @@ ENDIF()
 
 CONFIGURE_FILE(Config.h.in "${CMAKE_CURRENT_SOURCE_DIR}/Config.h")
 CONFIGURE_FILE(cuda/CUDAConfig.h.in "${CMAKE_CURRENT_SOURCE_DIR}/cuda/CUDAConfig.h")
+if(USE_ROCM)
```

Comment:
Do we still need the CUDAConfig.h when USE_ROCM=ON?

Reply:
I think there might be some CPU files that rely on it. I'll check in another PR.

Reply:
meh, rebasing, I'll check

Reply:
Yeah, we do need it. I added a comment about it. We can get rid of it when out-of-place hipify happens.